index.html

```diff
@@ -39,7 +39,7 @@ <h3 class="text-muted">Chromebrew</h3>
       </div>
 
       <div class="row marketing">
-        <div class="col-lg-6">
+        <div class="col-6">
```

Comment:
We can do another `col` size, I just did it this way because it was this way before.
But yes, `col-12` would make more sense.